lib/pipeline.js

```diff
@@ -126,26 +102,6 @@ class Pipeline extends Readable {
       return variables.set(variable.out(ns.p('name')).value, variable.out(ns.p('value')).value)
     }, new Map())
   }
-
-  static pipelineNode (definition, iri) {
-    let node
-
-    if (iri) {
-      node = clownface(definition, rdf.namedNode(iri.value || iri.toString()))
-    } else {
-      node = clownface(definition).has(ns.rdf('type'), ns.p('Pipeline'))
-    }
-
-    if (!node.term) {
-      throw new Error('expected an existing IRI or a single Pipeline class in definition')
-    }
-
-    return node
-  }
-
-  static create (definition, { iri, basePath, context, objectMode, variables } = {}) {
-    return new Pipeline(Pipeline.pipelineNode(definition, iri), { basePath, context, objectMode, variables })
-  }
```

Comment:
This is probably the most significant "unrelated" change.

First, I require to call `new Pipeline` from the new loader, so I changed the export.
This caused a circular dependency between this file and the said loader so I moved the factory methods to a separate file too. Note that the main exports are the same so no change for external code